src/components/ReportActionItem/MoneyRequestView.tsx

```diff
@@ -375,7 +393,10 @@ function MoneyRequestView({
                         }
                     />
                 )}
-                {canUseViolations && <ViolationMessages violations={getViolationsForField('receipt')} />}
+                {/* eslint-disable-next-line @typescript-eslint/prefer-nullish-coalescing */}
+                {!shouldShowReceiptEmptyState && !shouldShowMapOrReceipt && <View style={{marginVertical: 6}} />}
+                {shouldShowNotesViolations && <ReceiptAuditMessages notes={noticeTypeViolations} />}
+                <ViolationMessages violations={getViolationsForField('receipt')} />
```

Comment:
Don't we need to check for `canUseViolations` still?
```suggestion
                {canUseViolations  && <ViolationMessages violations={getViolationsForField('receipt')} />}
```